charts/opslevel/templates/opslevel/secret.yaml

```
@@ -81,7 +81,8 @@ data:
   GITHUB_APP_IDENTIFIER: '{{ required "please provide 'integrations.github.secret.appId'" .Values.integrations.github.secret.appId | b64enc }}'
   GITHUB_PRIVATE_KEY: '{{ required "please provide 'integrations.github.secret.privateKey'" .Values.integrations.github.secret.privateKey | b64enc }}'
   GITHUB_WEBHOOK_SECRET: '{{ required "please provide 'integrations.github.secret.webhookSecret'" .Values.integrations.github.secret.webhookSecret | b64enc }}'
-  GITHUB_INSTALL_URL: '{{ template "github.install.url" . | b64enc }}'
+stringData:
+  GITHUB_INSTALL_URL: '{{ template "github.install.url" . }}'
```

```suggestion
  GITHUB_INSTALL_URL: '{{ include "github.install.url" . | b64enc }}'
```
Have you tried this?